tinyyaml.lua

```diff
@@ -510,7 +510,7 @@ local function parseseq(line, lines, indent)
     end
     local rest = ssub(line, j+1)
 
-    if sfind(rest, '^[^\'\"%s]*:') then
+    if sfind(rest, '^[^\'\"%s]*:%s') then
```

Comment:
Is there RFC or defacto implementation requires the hash value to start with whitespace?

Reply:
Failed to go through official yaml spec, but tested on https://onlineyamltools.com/convert-yaml-to-json
![image](https://user-images.githubusercontent.com/34589752/122858916-1ebf5280-d2e9-11eb-8875-938c1f71c0b8.png)
![image](https://user-images.githubusercontent.com/34589752/122858943-2d0d6e80-d2e9-11eb-9296-a6cd8653ce02.png)